pkgs/applications/audio/i-score/default.nix

```diff
@@ -0,0 +1,85 @@
+{
+  boost,
+  cln,
+  cmake,
+  fetchgit,
+  gcc5,
+  ginac,
+  jamomacore,
+  kde5,
+  libsndfile,
+  ninja,
+  portaudio,
+  qtbase,
+  qtdeclarative,
+  qtimageformats,
+  qtsvg,
+  qttools,
+  qtwebsockets,
+  rtaudio,
+  stdenv
+}:
+
+stdenv.mkDerivation rec {
+  version = "v1.0.0-a67";
```

Comment:
Versions must begin with numbers (otherwise, the parser will interpret the "v" as being part of the package name).